webpack.config.js

```diff
@@ -0,0 +1,53 @@
+"use strict";
+const NodePolyfillPlugin = require("node-polyfill-webpack-plugin");
+
+/** @type {import("webpack").Configuration} */
+module.exports = {
+    mode: "none",
+    entry: {
+        no_unsanitize: ["./web.js"],
+    },
+    output: {
+        filename: "[name].js",
+        library: "[name]",
+        libraryTarget: "umd",
+        globalObject: "this",
+    },
+    module: {
+        rules: [
+            {
+                test: /\.m?js$/u,
+                loader: "babel-loader",
+                options: {
+                    presets: [
+                        ["@babel/preset-env", {
+                            debug: true, // ← to print actual browser versions
+
+                            /*
+               * We want to remove `transform-unicode-regex` convert because of https://github.com/eslint/eslint/pull/12662.
+               *
+               * With `>0.5%`, `@babel/preset-env@7.7.6` prints below:
+               *
+               *     transform-unicode-regex { "chrome":"49", "ie":"11", "safari":"5.1" }
+               *
+               * So this excludes those versions:
+               *
+               * - IE 11
+               * - Chrome 49 (2016; the last version on Windows XP)
+               * - Safari 5.1 (2011-2013; the last version on Windows)
+               */
+                            targets: ">0.5%, not chrome 49, not ie 11, not safari 5.1",
+                        }],
+                    ],
+                },
+            },
+        ],
+    },
+    plugins: [
+        new NodePolyfillPlugin(),
+    ],
```

Comment:
Node-polyfill-webpack-plugin doesn’t seem to be polyfilling anything in the file webpack is building from the no-unsanitized sources, at least not at the moment, which is not super surprising given that we don’t use any special from nodejs that isn’t available in the js environment that webbrowsers provides.

I'd suggest to remove it all together (both the addition to the npm dependencies and from this webpack config), and spare ourselves a new dependency if not really used at the moment (and eventually reconsider if we would ever need it).